### PR TITLE
Nativescript 7 Update

### PR DESCRIPTION
**Pull request by PhilippS93**

### CODE REVIEW

**Comment by Burgov** [comment]
@EddyVerbruggen any chance you can take a look at this?

**Comment by EddyVerbruggen** [comment]
@Burgov Did you get a chance to review this perhaps? I'd be happy to merge it but hardly have time to test it myself.

**Review by dnepromell — APPROVED** [review]

File: src/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"name": "nativescript-apple-sign-in",
-	"version": "1.1.0",
+	"name": "nativescript-apple-sign-in-enduco",
```
Comment (Burgov) [inline src/package.json:2]: you made some changes in this file that shouldn't go into the PR. all the references to enduco should be reverted. After doing that, i was able to run the demo app 👍

**Review by Burgov — CHANGES_REQUESTED** [review]

File: src/package.json (hunk 1 above)
Comment (PhilippS93) [reply src/package.json:2]: Perfect, thanks!

Comment (EddyVerbruggen) [reply src/package.json:2]: Yeah, please revert the ones in package.json as well. Feel free to add yourself to the authors section though.

**Comment by Burgov** [comment]
@PhilippS93 i just tried it out in our project after upgrading to NS7, but i'm getting this error when I tap my button:

Authorization failed: Error Domain=AKAuthenticationError Code=-7026 UserInfo={AKClientBundleID=<private>}
CONSOLE LOG: Unexpected error occurred The operation couldn\M-b\M^@\M^Yt be completed. (com.apple.AuthenticationServices.AuthorizationError error 1000.)

The dialog never shows up. I'm not sure if the issue sits in this PR or in our code... Maybe we can look at it together?

**Comment by Burgov** [comment]
Are you in the Nativescript Slack workspace?

**Comment by Burgov** [comment]
ah sorry, my bad. My app.entitlements file got lost during the update. Reinstated it and now it works.

@EddyVerbruggen 👍  all good

**Review by Burgov — APPROVED** [review]

**Comment by EddyVerbruggen** [comment]
Thanks a lot guys!

I've published 2.0.0 just now, please give it a spin.

I tested quickly on the simulator and all went well except for returning to the app after signing in - assuming it's a simulator issues and don't have time to check it on a device just now. It was also reported in #14, so was already an issue before the upgrade.

**Pull request merged**